ispdb/mnet.ne.jp.xml

```diff
@@ -4,18 +4,28 @@
     <domain>mnet.ne.jp</domain>
     <displayName>Mnet メール サービス</displayName>
     <displayShortName>Mnetメール</displayShortName>
+    <incomingServer type="imap">
+      <hostname>mail.mnet.ne.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILADDRESS%</username>
+      <authentication>password-cleartext</authentication>
+      <authentication>password-encrypted</authentication>
```

Comment:
If encrypted passwords work, please put them first. Otherwise, there's no point adding them, because the client will try the first auth method that it knows, so clients will always use cleartext.

Reply:
Changed in 295deaaeea2e93504a958da2b4000a62dad91726